cardano-cli/src/Cardano/CLI/EraBased/Run/CreateTestnetData.hs

```diff
@@ -225,14 +227,18 @@ runGenesisCreateTestNetDataCmd Cmd.GenesisCreateTestNetDataCmdArgs
 
   when (0 < numUtxoKeys) $ writeREADME utxoKeysDir utxoKeysREADME
 
-  let mayStakePoolRelays = Nothing -- TODO @smelc temporary?
+  mSPOsRelays <- forM relays readRelays
+  case (relays, mSPOsRelays) of
+    (Just fp, Just stakePoolRelays) | Map.size stakePoolRelays > fromIntegral numPools ->
+      throwError $ GenesisCmdTooManyRelaysError fp (fromIntegral numPools) (Map.size stakePoolRelays)
```

Comment:
This check wasn't in `create-staked`, but checked with @mgmeier that it was nice to have in `create-testnet-data`.

Reply:
Just curious, what's wrong with having more relays than SPOs?